src/SFA.DAS.Activities.sln

```diff
@@ -27,16 +25,16 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SFA.DAS.Activities.Acceptan
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SFA.DAS.Activities.IntegrationTests", "SFA.DAS.Activities.IntegrationTests\SFA.DAS.Activities.IntegrationTests.csproj", "{D1B06E53-A92E-4BA7-9472-A5F013B80987}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SFA.DAS.IntegrityChecker.Worker", "SFA.DAS.IntegrityChecker.Worker\SFA.DAS.IntegrityChecker.Worker.csproj", "{59DB7E32-5FA4-47B0-AC7B-D74FF3D67EDC}"
```

Comment:
Going with our new project naming conventions can this be renamed to `SFA.DAS.Activities.Jobs`?